lark-stubs/exceptions.pyi

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Iterable, Callable
+from .tree import Tree
+
+
+class LarkError(Exception):
```

Comment:
So these empty exceptions can be removed now, right?

Reply:
The public exception should be listed to avoid import problem in MyPy, because `pyi` will replace `py` module when doing type check.

Reply:
Why would there be an import problem though?

Reply:
For users checking their code. MyPy will check the annotated dependencies either.

Reply:
Would it be better to identify the public name if you add a `__all__` list? ([PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names))

Reply:
If it's necessary and will allow to remove these re-definitions, that I'll consider it.

Is the an import problem currently? I don't understand why..

Reply:
Do you mean that empty inheritance like "LarkError" should be removed? If it is a public name, it can't, and its existence is necessary. If not, other subclasses can inherit from `Exception` directly.

Reply:
My question is why do exceptions need to be included at all? There is no additional information presented in them, and they are already accessible by importing exceptions.py?

Reply:
No, pyi will replace py files if exist.

<https://mypy.readthedocs.io/en/stable/stubs.html#creating-a-stub>

> If a directory contains both a .py and a .pyi file for the same module, the .pyi file takes precedence.